engine/runtime/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -548,12 +567,46 @@ class Compiler(
     uncachedParseModule(module, isGenDocs)
   }
 
+  private val ieb: scala.collection.mutable.Map[Module, BindingsMap] =
```

Comment:
What does `ieb` stand for?

Reply:
"import export bindings". Anyway removed in 143e4f9 to narrow the scope of the PR down.